services/worker-manager/src/providers/provider.js

```diff
@@ -61,7 +61,7 @@ class Provider {
   async scanPrepare() {
   }
 
-  async checkWorker({Worker}) {
+  async checkWorker({Worker}) { // todo: it seems we're actually getting worker object
```

Comment:
yep, this should be a lower-case "w"

Reply:
this error comes from the google provider. I'll make a PR